src/shop/models.py

```diff
@@ -87,6 +97,15 @@ class Meta:
         verbose_name_plural = _('supplier products')
         unique_together = ('supplier', 'sku',)
 
+    @property
+    def unit_price(self):
+        return self.price / self.qty_multiplier
```

Comment:
It might be unlikely, but couldn't the multiplier end up as `0` here? I didn't see any barriers in the admin model.